tasks/run-local.sh

```diff
@@ -46,8 +81,12 @@ else
     chmod -R go+rX "$SECRETS"/webhook
 fi
 
+if [ -n "$TOKEN" ]; then
+    cp -fv "$TOKEN" "$SECRETS"/webhook/.config--github-token
+fi
+
 # start podman and run RabbitMQ in the background
-podman run -d --name cockpituous-rabbitmq --pod=new:cockpituous -v "$RABBITMQ_CONFIG":/etc/rabbitmq:ro -v "$SECRETS"/webhook:/run/secrets/webhook:ro docker.io/rabbitmq:3-management
+podman run -d --name cockpituous-rabbitmq --pod=new:cockpituous -v "$RABBITMQ_CONFIG":/etc/rabbitmq:ro -v "$SECRETS"/webhook:/run/secrets/webhook:ro -p 5671:5671 docker.io/rabbitmq:3-management
```

Comment:
Do we still need to expose the queue in that way? It now calls the scans inside the container, and for outside testing I think we really want to end JSON payloads to the webhook http.